mlir/lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -1640,6 +1640,11 @@ LogicalResult CppEmitter::emitType(Location loc, Type type) {
   }
   if (auto fType = dyn_cast<FloatType>(type)) {
     switch (fType.getWidth()) {
+    case 16:
+      if (llvm::isa<Float16Type>(type))
+        return (os << "_Float16"), success();
+      else
```

Comment:
Please explicitly check for bfloat16; at the pace of current floating point development, I wouldn't be surprised if we get another 16 bit float type in the future.

Reply:
Done